Task_DEV_3/Task_DEV_3/Program.cs

```diff
@@ -4,39 +4,49 @@ namespace Task_DEV_3
 {
     class Program
     {
+        static void InputArrayElement(int[] inputNumbers, int arrayLength)
+        {
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+        }
+        static void CheckingNumbersForPrime(int[] inputNumbers, int arrayLength)
+        {
+            bool isPrimeNumber;
+            for (int i = 0; i < arrayLength; i++)
+            {
+                isPrimeNumber = true;
+                for (int j = 2; j < inputNumbers[i]; j++)
+                {
+                    if (inputNumbers[i] % j == 0)
+                    {
+                        isPrimeNumber = false;
+                        break;
+                    }
+                }
+                if (isPrimeNumber)
+                {
+                    Console.WriteLine($"{inputNumbers[i]} is prime number");
```

Comment:
Single responsibility violation. If method checks smth it doesn't need to print smth to console.